index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>A Meaningful Page Title</title>
+    <link href="style.css" rel="stylesheet" />
+  </head>
+
+  <body>
+    <div id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <div id="img-div">
```

Comment:
- [ ] Elements wrapped inside `main` should use `section` tag, please change `div` to `section`